src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -890,6 +890,9 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_common_t, uct_iface_ops_t *tl_ops,
         if (status != UCS_OK) {
             goto cleanup_mpool;
         }
+    } else {
+        self->event_channel    = NULL;
+        self->cq_event_channel = NULL;
```

Comment:
I think we can remove this initialization (to NULL) from uct_rc_mlx5_devx_iface_init_events